graphblas_algorithms/nxapi/shortest_paths/weighted.py

```diff
@@ -0,0 +1,47 @@
+from graphblas_algorithms import algorithms
+from graphblas_algorithms.classes.digraph import to_graph
+
+from ..exception import NetworkXUnbounded, NodeNotFound
+
+__all__ = [
+    "all_pairs_bellman_ford_path_length",
+    "single_source_bellman_ford_path_length",
+]
+
+
+def all_pairs_bellman_ford_path_length(G, weight="weight", *, chunksize=128):
```

Comment:
We're adding a new keyword-only argument to `nxapi`. @jim22k, do you like `chunksize` for this?

We have a similar extra keyword-argument `nchunks` in `nxapi.square_clustering` that controls how much memory is used during calculation.

Neither one, `nchunks` or `chunksize`, changes the result; they're only used to control memory use and performance via available parallelism.